src/dropWhile.js

```diff
@@ -5,10 +5,10 @@ var _xdropWhile = require('./internal/_xdropWhile');
 
 
 /**
- * Returns a new list containing the last `n` elements of a given list, passing
- * each value to the supplied predicate function, skipping elements while the
- * predicate function returns `true`. The predicate function is passed one
- * argument: *(value)*.
+ * Returns a new list excluding the leading elements of a given list which
+ * match the supplied predicate function. It passes each value to the supplied
+ * predicate function, skipping elements while the predicate function returns
+ * `true`. The predicate function is passed one argument: *(value)*.
```

Comment:
I'd like to see "is applied to" in place of "is passed" (here and in the description of `dropLastWhile`).